packages/thyra/core/src/support/operator_vector/client_support/Thyra_VectorStdOpsTester_def.hpp

```diff
@@ -596,7 +596,37 @@ bool VectorStdOpsTester<Scalar>::checkStdOps(
         )
       ) success=false;
   }
-  
+
+// Test pair_wise_max
+  out << "\n"<<tc<<") Testing pair_wise_max(*v1, *v2, z.ptr()) ...\n";
+  ++tc;
+  {
+    v1  = createMember(vecSpc);
+    v2  = createMember(vecSpc);
+    v3  = createMember(vecSpc);
+    z   = createMember(vecSpc);
+    const Scalar alpha = as<Scalar>(2.0);
+    seed_randomize<Scalar>(12345);
+    randomize(as<Scalar>(-ST::one()),ST::one(),v1.ptr()); // v1 = rand
+    randomize(as<Scalar>(-ST::one()),ST::one(),v2.ptr()); // v2 = rand
+
+    // z = alpha*max(v1_i, v2_i)
+    pair_wise_max(alpha, *v1, *v2, z.ptr()); 
+
+    V_VpV(v3.ptr(), *v1, *v2); // v3 = v1 + v2
+    Vp_V(v1.ptr(), *v2, as<Scalar>(-ST::one())); // v1 -= v2
+    abs(*v1, v2.ptr()); // v2 = abs(v1)
+    Vp_V(v3.ptr(), *v2); // v3 += v2
+
+    // v1 + v2 + | v1 - v2 | = 2 max (v1, v2)
+    Vp_V(z.ptr(), *v3, as<Scalar>(-ST::one())); // z -= v3
+    if(!testMaxErr<Scalar>(
+         "norm_2(*z)",norm_2(*z)
+         ,"error_tol",error_tol(),"warning_tol",warning_tol(),&out
+        )
+      ) success=false;
+  }
```

Comment:
I am not sure I understand this test.  Another option is to just it to write a very specific test for this RTOp that uses manually set values and and a serial vector.  RTOps don't know anything about parallelization so it would be easy to write a specific unit test for it.  Then the Thyra-level test just needs to make sure that the vectors and scalar are passed in correctly.